locales/en.yml

```
@@ -63,6 +63,10 @@ events:
     concert: Concert
     show: Show
     other: Other
+  edit:
+    post: Post
+    event: Event
+    save: save
```

Original translation was different ;) 

```suggestion
    save: Save
```